<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background:rgb(255, 255, 255);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>4</x>
      <y>240</y>
      <width>791</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
border:1px solid rgb(84, 84, 84);
}</string>
    </property>
    <property name="placeholderText">
     <string>enter text and count word or typing speed and search text to...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>470</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>440</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(162, 162, 162);
}</string>
    </property>
    <property name="text">
     <string>search word</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>440</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(162, 162, 162);
}</string>
    </property>
    <property name="text">
     <string>words per min</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>470</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(80, 80, 80);
}</string>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>520</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>520</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>440</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
color:rgb(162, 162, 162);
}</string>
    </property>
    <property name="text">
     <string>00:00</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit_2">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>0</y>
      <width>791</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QTextEdit{
border:1px solid rgb(84, 84, 84);
}</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:10pt; font-weight:600; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'Roboto','sans-serif'; font-size:8pt; font-weight:400; color:#333333; background-color:#ffffff;&quot;&gt;Elephants are the worldâ€™s largest terrestrial mammals, commonly found in deep forests in Asia and Africa. Elephants are large, wild animals that live in herds. They are among the smartest and most social animals on the planet. They never leave their herds, and if one of them goes missing, they make every attempt to locate it. They are the ideal companions for people, yet they may also be harmful at times. Elephants are vegetarians and eat leaves, twigs, fruits, vegetables, etc. If they are tamed, they also eat bread, sugarcane, etc. Elephants are useful in various ways, like in wars, for lifting heavy objects, felling trees, etc. In ancient times, people used elephants in wars and also as a means of transportation. Elephants are tamed and trained by people and taken to the circuses for entertainment. Elephants are perfect companions to human beings and therefore, they listen to their owners with sincerity. In most places, elephants are considered cultural representations and are worshipped in many cultures. Elephant teeth are very costly and are used for various purposes like making ornaments, combs, etc. Therefore, people also hunt these wild animals for selfish reasons. Along with the tusks, the dung of the elephants is also beneficial, which contains seeds of different plants, hence helping the plants to spread. Humans have been illegally killing elephants for their benefit, especially for taking the tusks. Therefore, humans are the reason behind their extinction. With increasing deforestation, elephants have lost their habitat and often come to human settlements searching for food, hence creating huge destruction to society.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="textInteractionFlags">
     <set>Qt::NoTextInteraction</set>
    </property>
    <property name="placeholderText">
     <string>enter text and count word or typing speed and search text to...</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
